backend/data/final_data/programsProcessed.json

```diff
@@ -270,71 +270,88 @@
         "components": {
             "non_spec_data": [
                 {
-                    "type": "other",
```

Comment:
Might just me missing something, but how has programsProcessed.json added 300 lines without changing programs_processing.py?

Reply:
Ah actually maybe it's just because programsPureRaw.json has more stuff in it now - I'll figure it out later

Reply:
hm I changed programs_formatting.py, not sure why programsProcessed has changed